lms/static/sass/lms-course-rtl.scss

```diff
@@ -0,0 +1,11 @@
+@import 'bourbon/bourbon';
+@import 'vendor/bi-app/bi-app-rtl'; // set the layout for right to left languages
+
+@import 'base/reset';
+@import 'base/variables';
+@import 'base/font_face';
+@import 'base/mixins';
+
+//<THEME-OVERRIDE>
```

Comment:
This is to support the conditionals needed for old-style Stanford-style theming.  I'll add a comment indicating what it's there for.